spec/console_spec.rb

```diff
@@ -0,0 +1,182 @@
+RSpec.describe Console do
+  let(:game) { Game.new }
+  let(:storage) { Storage.new }
+  let(:hint) { 'Hint' }
+  let(:test_guess) { '1234' }
+  let(:agree_command) { 'Yes' }
+  let(:disagree_command) { 'No' }
+  let(:name) { 'Rostik' }
+  NAME_DIFFICULTIES_FOR_PUTS = { difficulties: 'easy, medium, hell' }.freeze
```

Comment:
9, 10, 11 line is duplication of your code in main app.
Use those constants. Remove it from here